web/src/beta/lib/core/engines/Cesium/hooks.ts

```diff
@@ -132,6 +132,66 @@ export default ({
     property?.light?.lightIntensity,
   ]);
 
+  // shadow map
+  type ShadowMapBias = {
+    polygonOffsetFactor: number;
+    polygonOffsetUnits: number;
+    normalOffsetScale: number;
+    normalShading: boolean;
+    normalShadingSmooth: number;
+    depthBias: number;
+  };
```

Comment:
Could you move this to here? https://github.com/reearth/reearth/blob/eb22f86a295fb32ce40fb785323ff99be8358b8b/web/src/beta/lib/core/engines/Cesium/types.ts